### ISSUE TITLE
globe doesn't show the data with the expected look

### ISSUE THREAD

**Issue by ShekharReddy4** [KMITFOSS/openmrs-webgl-globe#1]

**Comment by rishirajjain** [comment]
I'm interested in this, can you provide a more elaborate descriprion?

**Comment by ShekharReddy4** [comment]
@rishirajjain yeah sure... did u clone this? I din update the readme from the original fork :)

if you clone this and run it locally then you will get a basic idea of what the actual bug is.
Thanks for the word :)

**Comment by rishirajjain** [comment]
@ShekharReddy4 yes, cloned it and checking out the code. :)

**Comment by rishirajjain** [comment]
@ShekharReddy4  Trying to figure out how to solve the bug. I'm a beginner so it would be helpful if I could get some pointers. Thanks in advance :D

**Comment by ShekharReddy4** [comment]
@rishirajjain  ok cool.. I have added the contributing.md file to the repo jus now, you can pull the changes and I am hopeful that you will get some more Idea regarding the repo :-) Keep going...

**Comment by rishirajjain** [comment]
okay thanks.

**Comment by rishirajjain** [comment]
I can't seem to find how to solve this, the map is refusing to show up at all. This is what I'm getting,
![screenshot from 2016-10-06 23-10-06](https://cloud.githubusercontent.com/assets/20815882/19163571/2364761e-8c1a-11e6-9d5a-4b9e90c08cf5.png)

**Comment by ShekharReddy4** [comment]
@rishirajjain I don understand what is the exact problem but you should head to the url 0.0.0.0:8000/globe/
